auth/auth.go

```diff
@@ -59,7 +59,7 @@
 //
 // First create an Auth object:
 //
-//  myAuth := auth.New(tcclient.Credentials{ClientId: "myClientId", AccessToken: "myAccessToken"})
+//  myAuth := auth.New(&tcclient.Credentials{ClientId: "myClientId", AccessToken: "myAccessToken"})
```

Comment:
Are we referencing it by pointer now because the potential of authorized scopes being large and it would be inefficient passing around the struct?